L1Trigger/L1TGlobal/interface/L1TGlobalUtil.h

```diff
@@ -102,7 +104,7 @@ namespace l1t {
     // It is provided only until prescales and masks are available as CondFormats...
     // Most users should simply ignore this method and use the default ctor only!
     // Will look for prescale csv file in L1Trigger/L1TGlobal/data/Luminosity/startup/<filename>
-    void OverridePrescalesAndMasks(std::string filename, unsigned int psColumn = 1);
+    void OverridePrescalesAndMasks(std::string filename, double psColumn = 1.);
```

Comment:
if this is the index of the prescale column, rather than the prescale value, it should stay as an `unsigned int` ?

Reply:
correct. reverting the developer's change.